recipes/drupal7-subfolder/README.md

```diff
@@ -0,0 +1,207 @@
+Writing a public summary of what we had to do to make DDEV work with a Drupal 7 site subfolder setup.
+
+## Subfolder
+
+A Drupal site might be operated from a subfolder (subdirectory) of a domain. Example for such a URL:
+* No Subfolder frontpage: https://example.com
+* Subfolder frontpage: https://example.com/jobs
+
+That means the Drupal site only lives in the `/jobs` part. A reverse proxy sits in front of the site and delivers requests either to the main page "/" or the Drupal site at "/jobs.
+
+## Simulating a subfolder setup in DDEV
+
+In order to have a development environment that mimics the production site subfolder as much as possible we want to run DDEV also in a subfolder. We need to change a couple of things for that.
+
+### Nginx
+
+Nginx needs 2 rewrite rules for the `/jobs` subfolder (replace /jobs with your subfolder name). Create `.ddev/nginx/subfolder.conf` (see also https://ddev.readthedocs.io/en/stable/users/extend/customization-extendibility/#providing-custom-nginx-configuration )
+
+```
+# Rewrite the /jobs subfolder.
+# Special rule for the front page: we need to pass "/" explicitly here to
+# $_GET['q'] and PHP. This is not done by Nginx automatically and then
+# Drupal falls back to the full request URI which would include the
+# subfolder /jobs.
+rewrite ^/jobs/?$    /index.php?q=/&$args   last;
+rewrite ^/jobs/(.*)$   /$1   last;
```

Comment:
With the current layout, it would be easier to just have this rewrite in the full nginx-site.conf.

Reply:
yep, done!